src/frequenz/client/microgrid/_client.py

```diff
@@ -122,3 +125,31 @@ async def __aexit__(
                 "Error while disconnecting from the microgrid API", exceptions
             )
         return result
+
+    async def get_microgrid_info(  # noqa: DOC502 (raises ApiClientError indirectly)
+        self,
+    ) -> MicrogridInfo:
+        """Fetch the information about the local microgrid.
+
+        This consists of information that describes the overall microgrid, as opposed to
+        its electrical components or sensors, e.g., the microgrid ID, location.
```

Comment:
Is this from the API?  if not, I'd rephrase, for example, like:

```suggestion
        """Retrieve information about the local microgrid.

        This consists of information about the overall microgrid, for example, the
        microgrid ID and its location.  It does not include information about the
        electrical components or sensors in the microgrid.

Reply:
[Most of it](https://github.com/frequenz-floss/frequenz-api-microgrid/blob/v0.17.x/proto/frequenz/api/microgrid/v1/microgrid.proto#L27-L30), I just adapted it to the Python style.

I will apply it, but it would be awesome if you can create issues or PRs in the API specs to improve the docs, or maybe we just want to do it all in one go for 1.0, not sure if @tiyash-basu-frequenz has a plan for it.